doc/source/ecosystem.rst

```diff
@@ -325,6 +325,17 @@ dimensional arrays, rather than the tabular data for which pandas excels.
 Out-of-core
 -------------
 
+`BCPandas <https://github.com/yehoshuadimarsky/bcpandas>`__
```

Comment:
I would make a new section about IO, as this doesn't really fit for "out-of-core" ? 
You can eg put it just above "out of core"